### ISSUE TITLE
HK Red MultiWii PRO Calibration problem

### ISSUE THREAD

**Issue by Nakedhoof** [MegaPirateNG/ardupilot-mpng#135]
I'm not sure if this is a software issue, or a hardware problem. Every time I initialize my FC, my compass heading on the pitch (y) axis is 32 degrees into the dirt. I can compensate for this error in the advanced parameters by setting INS_GYROFFS_Y to -0.321, but it gets set back to 32 degrees down whenever it coldstarts. Every time I try to calibrate, it fails. When I go to the compass calibration, I get no data points. I'm thinking it's the board.

**Comment by Nakedhoof** [comment]
After careful examination of the board with a microscope, I found a solder bridge. Once removed, I got data points on calibration, but still ended up pointing into the ground 32 degrees. I didn't examine the entire board. There might be more bridges.

**Comment by Nakedhoof** [comment]
Okay, I got this sussed. Made a test liftoff today, and she's a bit twitchy, but all looks to be working, and I was able to hover indoors without breaking anything for about 10-15 seconds, and I have tight spaces here

The command to solve this was in terminal-->setup-->level. Once this is performed, it is persistent.

I still have to disable checks in order to arm, as the accelerometer tests returns a fail.
